lib/providers/player/audio_player_provider.dart

```diff
@@ -33,13 +33,17 @@ class AudioPlayerNotifier extends BaseAudioHandler
 
   @override
   Future<void> play() async {
-    var checkBgAudio = mediaItemHasBGSound();
-    if (checkBgAudio) {
-      playBackgroundSound();
-    } else {
-      pauseBackgroundSound();
+    try {
+      unawaited(trackAudioPlayer.play());
+      var checkBgAudio = mediaItemHasBGSound();
+      if (checkBgAudio) {
+        playBackgroundSound();
+      } else {
+        pauseBackgroundSound();
+      }
+    } catch (e) {
+      print(e.toString());
```

Comment:
This isn't really helpful in prod, should we do a sentry call here?

Reply:
Oh yeah sure